### PR TITLE
MAB FIxes: Updates MOPAK port on profiler moorings, cleans up SM sheets

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP10CNSM/CP10CNSM_D00001_ingest.csv [hunk 1]
```diff
@@ -29,7 +29,7 @@ mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data
...
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl11/syslog/*.syslog.log,CP10CNSM-SBD11-00-DCLENG000,telemetered,Available,
-mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl11/mopak/*.mopak.log,CP10CNSM-SBD11-01-MOPAK0000,telemetered,Available,"Will only be ingested for burn-in, we do not ingest telem anymore"
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl11/mopak/*.mopak.log,CP10CNSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will not be telemetered
 mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl11/hyd1/*.hyd1.log,CP10CNSM-SBD11-02-HYDGN0000,telemetered,Available,
 mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl11/spkir1/*.spkir1.log,CP10CNSM-SBD11-05-SPKIRB000,telemetered,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl11/metbk1/*.metbk1.log,CP10CNSM-SBD11-06-METBKA000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP10CNSM/CP10CNSM_D00001_ingest.csv:35]: updates look good

File: CP11NOSM/CP11NOSM_D00001_ingest.csv [hunk 2]
```diff
@@ -28,7 +28,7 @@ mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/who
...
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00001/cg_data/dcl11/syslog/*.syslog.log,CP11NOSM-SBD11-00-DCLENG000,telemetered,Available,
-mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00001/cg_data/dcl11/mopak/*.mopak.log,CP11NOSM-SBD11-01-MOPAK0000,telemetered,Available,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00001/cg_data/dcl11/mopak/*.mopak.log,CP11NOSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will not be telemetered
 mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00001/cg_data/dcl11/hyd1/*.hyd1.log,CP11NOSM-SBD11-02-HYDGN0000,telemetered,Available,
 mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00001/cg_data/dcl11/spkir1/*.spkir1.log,CP11NOSM-SBD11-05-SPKIRB000,telemetered,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11NOSM/D00001/cg_data/dcl11/metbk/*.metbk.log,CP11NOSM-SBD11-06-METBKA000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP11NOSM/CP11NOSM_D00001_ingest.csv:34]: updates look good

File: CP11SOSM/CP11SOSM_D00001_ingest.csv [hunk 3]
```diff
@@ -28,7 +28,7 @@ mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/who
...
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl11/syslog/*.syslog.log,CP11SOSM-SBD11-00-DCLENG000,telemetered,Available,
-mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl11/mopak/*.mopak.log,CP11SOSM-SBD11-01-MOPAK0000,telemetered,Available,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl11/mopak/*.mopak.log,CP11SOSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will not be telemetered
 mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl11/hyd1/*.hyd1.log,CP11SOSM-SBD11-02-HYDGN0000,telemetered,Available,
 mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl11/spkir1/*.spkir1.log,CP11SOSM-SBD11-05-SPKIRB000,telemetered,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl11/metbk/*.metbk.log,CP11SOSM-SBD11-06-METBKA000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP11SOSM/CP11SOSM_D00001_ingest.csv:34]: looks good

File: CP13EAPM/CP13EAPM_D00001_ingest.csv [hunk 4]
```diff
@@ -1,12 +1,12 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/imm/adcpt/adcpt*.DAT,CP13EAPM-RII01-02-ADCPTG010,telemetered,Available,
 mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/irid2shore/stc_status*.txt,CP13EAPM-SBS01-00-STCENG000,telemetered,Available,
-mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/3dmgx3/*3dmgx3.log,CP13EAPM-SBS01-01-MOPAK0000,telemetered,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/3dmgx3/*3dmgx3.log,CP13EAPM-SBS01-05-MOPAK0000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP13EAPM/CP13EAPM_D00001_ingest.csv:4]: refdes update looks good

File: CP13NOPM/CP13NOPM_D00001_ingest.csv [hunk 5]
```diff
@@ -1,7 +1,7 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP13NOPM/D00001/imm/adcpt/adcpt*.DAT,CP13NOPM-RII01-02-ADCPTG010,telemetered,Available,
 mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP13NOPM/D00001/irid2shore/stc_status*.txt,CP13NOPM-SBS01-00-STCENG000,telemetered,Available,
-mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP13NOPM/D00001/3dmgx3/*3dmgx3.log,CP13NOPM-SBS01-01-MOPAK0000,telemetered,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP13NOPM/D00001/3dmgx3/*3dmgx3.log,CP13NOPM-SBS01-05-MOPAK0000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP13NOPM/CP13NOPM_D00001_ingest.csv:4]: refdes update looks good

File: CP13SOPM/CP13SOPM_D00001_ingest.csv [hunk 6]
```diff
@@ -1,7 +1,7 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP13SOPM/D00001/imm/adcpt/adcpt*.DAT,CP13SOPM-RII01-02-ADCPTG010,telemetered,Available,
 mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP13SOPM/D00001/irid2shore/stc_status*.txt,CP13SOPM-SBS01-00-STCENG000,telemetered,Available,
-mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP13SOPM/D00001/3dmgx3/*3dmgx3.log,CP13SOPM-SBS01-01-MOPAK0000,telemetered,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP13SOPM/D00001/3dmgx3/*3dmgx3.log,CP13SOPM-SBS01-05-MOPAK0000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP13SOPM/CP13SOPM_D00001_ingest.csv:4]: refdes update looks good

File: CP14NEPM/CP14NEPM_D00001_ingest.csv [hunk 7]
```diff
@@ -1,7 +1,7 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP14NEPM/D00001/imm/adcps/adcps*.DAT,CP14NEPM-RII01-02-ADCPSL010,telemetered,Available,
 mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP14NEPM/D00001/irid2shore/stc_status*.txt,CP14NEPM-SBS01-00-STCENG000,telemetered,Available,
-mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP14NEPM/D00001/3dmgx3/*3dmgx3.log,CP14NEPM-SBS01-01-MOPAK0000,telemetered,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP14NEPM/D00001/3dmgx3/*3dmgx3.log,CP14NEPM-SBS01-05-MOPAK0000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP14NEPM/CP14NEPM_D00001_ingest.csv:4]: refdes update looks good

File: CP14SEPM/CP14SEPM_D00001_ingest.csv [hunk 8]
```diff
@@ -1,7 +1,7 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP14SEPM/D00001/imm/adcps/adcps*.DAT,CP14SEPM-RII01-02-ADCPSL010,telemetered,Available,
 mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP14SEPM/D00001/irid2shore/stc_status*.txt,CP14SEPM-SBS01-00-STCENG000,telemetered,Available,
-mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP14SEPM/D00001/3dmgx3/*3dmgx3.log,CP14SEPM-SBS01-01-MOPAK0000,telemetered,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP14SEPM/D00001/3dmgx3/*3dmgx3.log,CP14SEPM-SBS01-05-MOPAK0000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP14SEPM/CP14SEPM_D00001_ingest.csv:4]: refdes update looks good

**Review by rtravis-whoi — APPROVED** [review]

File: CP10CNSM/CP10CNSM_D00001_ingest.csv (hunk 1 above)
Comment (reedan88) [inline CP10CNSM/CP10CNSM_D00001_ingest.csv:35]: Looks good.

File: CP11NOSM/CP11NOSM_D00001_ingest.csv (hunk 2 above)
Comment (reedan88) [inline CP11NOSM/CP11NOSM_D00001_ingest.csv:34]: Looks good.

File: CP11SOSM/CP11SOSM_D00001_ingest.csv (hunk 3 above)
Comment (reedan88) [inline CP11SOSM/CP11SOSM_D00001_ingest.csv:34]: Looks good.

File: CP13EAPM/CP13EAPM_D00001_ingest.csv [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
 mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/imm/mmp/C*.DAT,CP13EAPM-WFP01-02-DOFSTK000,telemetered,Available,
 mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/imm/mmp/C*.DAT,CP13EAPM-WFP01-03-CTDPFK000,telemetered,Available,
 mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/imm/mmp/E*.DAT,CP13EAPM-WFP01-04-FLORTK000,telemetered,Available,
 mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/imm/mmp/E*.DAT,CP13EAPM-WFP01-05-PARADK000,telemetered,Available,
 mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_telemetered_driver,/omc_data/whoi/OMC/CP13EAPM/D00001/imm/ctdmo/ctdmo*.DAT,CP13EAPM-SBI01-02-CTDMOS011,telemetered,Available,
-#,,CP13EAPM-RII01-02-ADCPUP021,telemetered,Not Expected,No inductive data expected for this deployment
+#,,CP13EAPM-RII01-02-ADCPUP021,telemetered,Not Expected,No inductive data expected for this deployment
```
Comment (reedan88) [inline CP13EAPM/CP13EAPM_D00001_ingest.csv:12]: Looks good.

File: CP13NOPM/CP13NOPM_D00001_ingest.csv [hunk 10]
```diff
@@ -1,7 +1,7 @@
...
 mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP13NOPM/D00001/imm/mmp/E*.DAT,CP13NOPM-WFP01-00-WFPENG001,telemetered,Available,
 mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP13NOPM/D00001/imm/mmp/A*.DEC,CP13NOPM-WFP01-01-VEL3DK000,telemetered,Available,
 mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP13NOPM/D00001/imm/mmp/C*.DAT,CP13NOPM-WFP01-02-DOFSTK000,telemetered,Available,
```
Comment (reedan88) [inline CP13NOPM/CP13NOPM_D00001_ingest.csv:7]: Looks good.

File: CP13SOPM/CP13SOPM_D00001_ingest.csv [hunk 11]
```diff
@@ -1,7 +1,7 @@
...
 mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP13SOPM/D00001/imm/mmp/E*.DAT,CP13SOPM-WFP01-00-WFPENG001,telemetered,Available,
 mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP13SOPM/D00001/imm/mmp/A*.DEC,CP13SOPM-WFP01-01-VEL3DK000,telemetered,Available,
 mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP13SOPM/D00001/imm/mmp/C*.DAT,CP13SOPM-WFP01-02-DOFSTK000,telemetered,Available,
```
Comment (reedan88) [inline CP13SOPM/CP13SOPM_D00001_ingest.csv:7]: Looks good.

File: CP14NEPM/CP14NEPM_D00001_ingest.csv [hunk 12]
```diff
@@ -1,7 +1,7 @@
...
 mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP14NEPM/D00001/imm/mmp/E*.DAT,CP14NEPM-WFP01-00-WFPENG001,telemetered,Available,
 mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP14NEPM/D00001/imm/mmp/A*.DEC,CP14NEPM-WFP01-01-VEL3DK000,telemetered,Available,
 mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP14NEPM/D00001/imm/mmp/C*.DAT,CP14NEPM-WFP01-02-DOFSTK000,telemetered,Available,
```
Comment (reedan88) [inline CP14NEPM/CP14NEPM_D00001_ingest.csv:7]: Looks good.

File: CP14SEPM/CP14SEPM_D00001_ingest.csv [hunk 13]
```diff
@@ -1,7 +1,7 @@
...
 mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP14SEPM/D00001/imm/mmp/E*.DAT,CP14SEPM-WFP01-00-WFPENG001,telemetered,Available,
 mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP14SEPM/D00001/imm/mmp/A*.DEC,CP14SEPM-WFP01-01-VEL3DK000,telemetered,Available,
 mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP14SEPM/D00001/imm/mmp/C*.DAT,CP14SEPM-WFP01-02-DOFSTK000,telemetered,Available,
```
Comment (reedan88) [inline CP14SEPM/CP14SEPM_D00001_ingest.csv:7]: Looks good.

**Review by reedan88 — APPROVED** [review]
Updates look good.

**Pull request merged**